lms/djangoapps/course_api/blocks/views.py

```diff
@@ -170,6 +170,8 @@ class BlocksView(DeveloperErrorViewMixin, ListAPIView):
           * lti_url: The block URL for an LTI consumer. Returned only if the
             "ENABLE_LTI_PROVIDER" Django settign is set to "True".
 
+          * due: The due date of the block. Returned only if "due" is included
```

Comment:
@catong FYI on this update to the Course Blocks API - added support for the `due` date field.